automatic/chromium/tools/chocolateyInstall.ps1

```diff
@@ -14,7 +13,7 @@ $validExitCodes = @(0)
   if (Test-Path $Chromium) {
     $silentArgs = ''
   } else {
-    $silentArgs = '--system-level --do-not-launch-chrome'
+    $silentArgs = '--system-level'
   }
   
-    Install-ChocolateyPackage $packageName $fileType $silentArgs $url $validExitCodes $checksum $checksumType
+    Install-ChocolateyPackage $packageName $fileType $silentArgs $url $checksum $checksumType

```

Comment:
@ferventcoder I think I follow you Rob. Specifying the variable will cause chocolatey not too assume what is being passed.

Install-ChocolateyPackage $packageName $fileType $silentArgs $url -validExitCodes $validExitCodes -checksum $checksum -checksumType $checksumType

Should be my last line in the script, Right?